application/models/TemplateConfiguration.php

```diff
@@ -1354,4 +1380,188 @@ public function setOptionKeysToInherit(){
             $this->options = json_encode($aOptions);
         }
     }
+
+    /**
+     * Returns the list theme's file categories
```

Comment:
Returns a list of file categories for the theme.
Each category is related to a directory which holds files for the theme.
This files are usually listed to be selected as values for options.